### PR TITLE
Ch41/video crud endpoints for videos

### PR DESCRIPTION
**Pull request by AlineRibeiro**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    resources :videos, only: %i[index show destroy create]
+  end
```
Comment (guialbuk) [inline config/routes.rb]: `namespace` is also not only adding the `API::` namespace to the controller, that we want but also adds `/api` to all paths.

```bash
> rails routes | grep videos
     api_videos GET    /api/videos(.:format)                                                                             api/videos#index
                POST   /api/videos(.:format)                                                                             api/videos#create
     api_video GET     /api/videos/:id(.:format)                                                                         api/videos#show
                DELETE /api/videos/:id(.:format)                                                                         api/videos#destroy
```

We won't use `/api` in our paths. The [documentation](https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing) has an example just after the table on adding a namespace to the controller without adding it to the path.

File: spec/controllers/videos_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
```
Comment (guialbuk) [inline spec/controllers/videos_controller_spec.rb:1]: rubocop just  reminded me that the path should be `spec/controllers/api/videos_controller_spec.rb`, mirroring the path in `app/controllers/api/...`

**Review by guialbuk — COMMENTED** [review]
Please run `bundle exec rubocop -A` to auto-correct all files. We can through warnings that can't be auto-corrected to and check if they make sense or we should disable them.

File: config/routes.rb (hunk 1 above)
Comment (AlineRibeiro) [reply config/routes.rb]: If we are not using api in our paths, I dont see a reason to keep the api namespace. I'd rather keep it as /videos only, then, and make things simpler.

File: spec/controllers/videos_controller_spec.rb (hunk 2 above)
Comment (AlineRibeiro) [reply spec/controllers/videos_controller_spec.rb:1]: Not anymore, as we are not going to use the api namespace anymore. Thanks for the suggestion, though.

**Pull request merged**